### PR TITLE
Reader: A/B various height limits on featured images.

### PR DESCRIPTION
**Pull request by blowery**
Adds in a new A/B test that can try out various limits on featured image height. Doesn't affect embeds, as they're much harder to work with and predict with max heights like these.

Also changes A/B test a bit to allow folks that are already in an A/B test to continue being in that test, even if they're not eligible. This lets us debug A/B tests that most devs are not eligible for.

### CODE REVIEW

**Comment by blowery** [comment]
cc @jancavan

**Comment by blowery** [comment]
to get yourself into a test, you need to run the following in the console:

`localStorage.setItem('ABTests','{"readerShorterFeatures_20160311":"twenty"}')`

then refresh. valid values are `original`, `fifty`, `twentyfive`, `twenty`

**Comment by blowery** [comment]
original
screenshot

fifty
screenshot

twentyfive
screenshot

twenty
screenshot

**Comment by jancavan** [comment]
Much better! Between 20 and 25, their differences aren't very stark. What do you think about cutting the options down to three (original, 50 and, 20)?

**Comment by blowery** [comment]
That works. I think I'd take 25 over 20 as that's my personal favorite. :)

**Comment by jancavan** [comment]
That works for me :+1:

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+4.3.0
```
Comment (bluefuton) [inline .nvmrc]: I know it's tiny, but as it's not related to the A/B test, best off in a separate PR perhaps?

**Comment by bluefuton** [comment]
Works well :+1:

50 is my favourite :)

File: .nvmrc (hunk 1 above)
Comment (blowery) [inline .nvmrc]: bah. slipped in.

**Pull request merged**